app/mainAppWindow/index.js

```diff
@@ -323,12 +328,16 @@ function setImgSrcSecurityPolicy(policies) {
  * @param {Electron.BeforeSendResponse} callback
  */
 function onBeforeSendHeadersHandler(detail, callback) {
-	if (detail.url.startsWith(customBGServiceUrl.href)) {
-		detail.requestHeaders['Access-Control-Allow-Origin'] = '*';
+	if (intune.isSsoUrl(detail.url)) {
+		intune.addSsoCookie(logger, detail, callback);
+	} else {
+		if (detail.url.startsWith(customBGServiceUrl.href)) {
```

Comment:
this could have just been an } else if (...) {

Reply:
not a problem. I was thinking to maybe pull the callback call that happens in the intune/index.js in here (so we call it always), and always call it as it used to be before, but it will probably make the other code more complex so ignore the if else option. I do wonder what will happen if we don't do the callback. We can refactor in the near future but I agree the implementation hasn't change for the previous use case.